internal/envstest/testconfig/random.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package envstest
```

Comment:
I think random should still be in the `envstest` package, no?

Reply:
random is only called to seed random HMACs in the serverconfig

Reply:
I think it's called in integration.go too, but that's a different build tag, so you might not see the compilation error?

Either way, I think envstest.Random() is better than testconfig.Random()

Reply:
envstest.NewServer is the only external reference. I've moved random() to package scope for good measure